theano/gof/fg.py

```diff
@@ -16,7 +15,6 @@
 from theano import config
 
 from six import iteritems, itervalues
-from six.moves import StringIO
```

Comment:
These changes actually make flake8 fail. This should probably go in a different PR.